Source/WebKit/UIProcess/API/gtk/WebKitInputMethodContextImplGtk.cpp

```diff
@@ -164,6 +164,8 @@ static void webkitInputMethodContextImplGtkGetPreedit(WebKitInputMethodContext*
         }
     }
 
+    g_clear_pointer(&attrList, pango_attr_list_unref);
```

Comment:
so this clears the list object, but do we need to clear / release the list items? I understand from the code that we create a new object out from them, but that doesn't seem to delete them

Reply:
https://docs.gtk.org/gtk4/method.IMContext.get_preedit_string.html

Reply:
https://docs.gtk.org/Pango/method.AttrList.unref.html

Reply:
Thanks, Carlos!